functional-tests/src/test/resources/values/helm-values-coh-user-artifacts-mainClass.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Oracle Corporation and/or its affiliates.  All rights reserved.
+
+cluster: MyCoherenceCluster
+
+store:
+  cacheConfig: "server-cache-config.xml"
+  mainClass:   "com.tangosol.net.DefaultCacheServer"
```

Comment:
Did you mean to be testing the existing main in coherence.jar here or should the above be
"custom.ServerMain"?   The above is the default for store.mainClass.

Reply:
I tested custom.ServerMain manually.  I can see from the log that the server is started.  However, the readinessProbe was not successful for some reason.  Since the test is to ensure user can supply a main class, not the main class itself, I switched to just using com.tangosol.net.DefaultCacheServer.

Reply:
fixed